contracts/DAOTreasury.cdc

```diff
@@ -116,10 +103,13 @@ pub contract DAOTreasury {
 
     // Deposit tokens //
     pub fun depositTokens(identifier: String, vault: @FungibleToken.Vault) {
-      emit DepositTokens(identifier: identifier)
+      if self.vaults[identifier] != nil {
+        self.vaults[identifier]?.deposit!(from: <- vault)
+      } else {
+        self.vaults[identifier] <-! vault
```

Comment:
in the case there isn't already a vault here, we should throw an error.  maybe add an assert to check this at the beginning of this function

Reply:
good catch!

Reply:
But if we throw an error there will be no possibility to add a vault with a different identifier?

Reply:
Fixed!